app/gui2/src/components/ComponentBrowser.vue

```diff
@@ -107,26 +114,23 @@ const originScenePos = computed(() => {
 
 function panIntoView() {
   const origin = originScenePos.value
-  const inputArea = new Rect(
+  const screenRect = cbRoot.value?.getBoundingClientRect()
+  if (!screenRect) return
+  const area = new Rect(
     origin,
-    new Vec2(WIDTH, INPUT_AREA_HEIGHT).scale(clientToSceneFactor.value),
+    new Vec2(screenRect.width, screenRect.height).scale(clientToSceneFactor.value),
```

Comment:
```suggestion
    screenRect.size.scale(clientToSceneFactor.value),
```

Reply:
Well, Actually this is DOMRect, so it does not have "size" property.